README.md

```diff
@@ -65,30 +65,30 @@ This means that Linux users receive pre-release builds earlier, but Upscayl itse
     <img src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" height="50px"/>
   </a>
 
-Upscayl should be available on the software listings of most Linux operating systems. Your distro's Store app might also support the [Flatpak](https://flatpak.org/setup) or Snap version.
+Upscayl should be available in the software listings of most Linux operating systems. Your distro's Store app might also support the [Flatpak](https://flatpak.org/setup) or Snap versions.
```

Comment:
A software listing is like a document so on is probably better but in is also fine. We only have one snap version though.
```suggestion
Upscayl should be available in the software listings of most Linux operating systems. Your distro's Store app might also support the [Flatpak](https://flatpak.org/setup) or Snap version.
```